document-store/src/integrationTest/java/org/hypertrace/core/documentstore/mongo/MongoDocStoreTest.java

```diff
@@ -548,4 +548,67 @@ public void testCount() throws IOException {
   public void testSearch() throws IOException {
     DocStoreTest.testSearch(datastore, DocStoreTest.MONGO_STORE);
   }
+
+  @Test
+  public void testContains() throws IOException {
```

Comment:
Can it be a common test?

Reply:
No, implementation of `contains` is pending from `postgres` side.

Reply:
I see that it will be supported in subsequent change.